docs/activation.md

```diff
@@ -0,0 +1,34 @@
+# The `activation` ping
+
+## Description
+This ping is intended to provide a measure of the activation of mobile products.
```

Comment:
```suggestion
This ping provides a measure of the activation of mobile products.
```

Reply:
"intended to" sounds so passive. It is sent and it is as reliable as other pings.